base/server/src/test/java/com/netscape/cmscore/test/CMSBaseTestHelper.java

```diff
@@ -8,42 +8,24 @@
 import com.netscape.cmscore.dbs.DBSSession;
 import com.netscape.cmscore.dbs.DBSubsystem;
 
-import junit.framework.TestCase;
-
 /**
- * The base class for all CMS unit tests. This sets up some basic stubs
+ * Helpe class for CMS unit tests. This sets up some basic stubs
```

Comment:
Typo?

Reply:
Yeah, will fix this and the Sonar issues and merge, thanks!